pylmod/gradebook.py

```diff
@@ -948,3 +1024,152 @@ def spreadsheet2gradebook(
             csv_reader, email_field, non_assignment_fields
         )
         return response
+
+    def get_staff(self, gradebook_id, simple=False):
+        """Get staff list for gradebook.
+
+        Get staff list for the gradebook specified.
+
+        If simple=True, return a list of dictionaries, one dictionary
```

Comment:
@pdpinch @carsongee An example would be good idea, but a better question would be, "Which of these member attributes belong in a 'simple' list?".  I'm currently returning all of them but Shirley that is more than anyone needs.